pom.xml

```diff
@@ -124,6 +105,14 @@
                         </goals>
                     </execution>
                 </executions>
+                <configuration>
```

Comment:
I think I know what you are doing here but I'm very familiar with these options and I'm also not seeing any change in the generated javadocs with these configurations

I did however find the actual links for the versions being used.

https://docs.spring.io/spring-security/site/docs/5.6.1/api/

and

https://javadoc.io/doc/io.github.openfeign/feign-core/11.7/index.html

Reply:
i did some more digging and figured out what this does. the following links to the exact docs for the versions being used

```xml
<links>
    <link>https://docs.spring.io/spring-security/site/docs/5.6.1/api</link>
    <link>https://javadoc.io/doc/io.github.openfeign/feign-core/11.7</link>
</links>
```